Improve field border contrast ratio (#638)
seek-oss/braid-design-system: .changeset/brown-paws-brush.md

---
'braid-design-system': patch
---

Toggle: Hide border on dark backgrounds

> Following the pattern of other bordered elements and hiding the border when on dark backgrounds. Reduces visual noise and cleans results in cleaner lines.

Reviewer: ```suggestion
Following the pattern of other bordered elements, we now hide the border when on dark backgrounds. This reduces visual noise and results in cleaner lines.
```
